src/Symfony/Component/OptionsResolver/OptionConfigurator.php

```diff
@@ -134,4 +134,16 @@ public function info(string $info): static
 
         return $this;
     }
+
+    /**
+     * Sets whether ignore undefined options.
+     *
+     * @return $this
+     */
+    public function setIgnoreUndefined(bool $ignore = true): static
```

Comment:
Here is okay `ignoreUndefined` name, because this is the fluent configurator. My previous/last feedback was about the OptionResolver class. Sorry if it wasn't clear enough.